lib/widget/dropdown.dart

```diff
@@ -182,70 +212,194 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
     }
   }
 
-
-
   @override
   Widget buildWidget(BuildContext context) {
-    if (widget.getType() == SelectOneType.dropdown) {
+    if (widget._controller.autoComplete == false &&
+        widget.getType() == SelectOneType.dropdown) {
```

Comment:
need consistency checking if the widget is of type SelectOneType.dropdown. You should put the if check, then inside branch whether it's an AutoComplete or simle Dropdown.

Either that or just remove the SelectOneType check. Whoever implements another SelectOne widget will have to refactor this code anyway